src/eval.rs

```diff
@@ -128,6 +129,9 @@ pub struct MiriConfig {
     pub report_progress: Option<u32>,
     /// Whether Stacked Borrows retagging should recurse into fields of datatypes.
     pub retag_fields: bool,
+    /// The location of a shared object file to load when calling external functions
+    /// TODO! consider allowing users to specify paths to multiple SO files, or to a directory
```

Comment:
We generally use `FIXME` instead of `TODO` as `TODO` blocks merging in the rustc repo. I don't know if this is true for submodules, but let's be consistent